pyuvsim/uvsim.py

```diff
@@ -217,16 +218,14 @@ def uvdata_to_task_iter(task_ids, input_uv, catalog, beam_list, beam_dict):
         yield task
 
 
-def init_uvdata_out(uv_in, source_list_name, uvdata_file=None,
+def init_uvdata_out(uv_in, source_list_name,
```

Comment:
It might be worth keeping these separate in anticipation of future changes. The `uv_in` object gets broadcast and will probably need to be compressed or modified in other ways. We may even want to use a different object for carrying simulation parameters to separate ranks, which would replace `uv_in`.

Reply:
I see, ok.